<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetWorkWidget</class>
 <widget class="QWidget" name="NetWorkWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>674</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutConnect">
         <item>
          <widget class="QLabel" name="ConnectNumLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(197, 197, 197);</string>
           </property>
           <property name="text">
            <string>活动客户端数</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QWidget" name="LinkingNumwidget" native="true">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutMaxTimeOut">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="MaxTimeOutLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="layoutDirection">
            <enum>Qt::LeftToRight</enum>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 170, 0);
</string>
           </property>
           <property name="text">
            <string>最大响应时间</string>
           </property>
           <property name="textFormat">
            <enum>Qt::PlainText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MaxTimeOutText">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>---/ms</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutMinTimeOut">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="MinTimeOutLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 170, 0);
</string>
           </property>
           <property name="text">
            <string>最小响应时间</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MinTimeOutText">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>---/ms</string>
           </property>
           <property name="textFormat">
            <enum>Qt::AutoText</enum>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayoutAverage">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="AverageTimeOutLabel">
           <property name="font">
            <font>
             <pointsize>10</pointsize>
             <weight>75</weight>
             <italic>true</italic>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 170, 0);
</string>
           </property>
           <property name="text">
            <string>平均响应时间</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="AverageTimeOutText">
           <property name="font">
            <font>
             <pointsize>20</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">color:rgb(255, 255, 255);
</string>
           </property>
           <property name="text">
            <string>---/ms</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignHCenter|Qt::AlignTop</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="leftMargin">
        <number>4</number>
       </property>
       <property name="topMargin">
        <number>4</number>
       </property>
       <property name="rightMargin">
        <number>4</number>
       </property>
       <property name="bottomMargin">
        <number>4</number>
       </property>
       <item>
        <widget class="QPushButton" name="ThreadPoolButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{   
    
	border-image: url(:/image/Ico/ThreadPoolbuttonStop.ico);
    background:transparent; 
}
QPushButton:hover
{
    
	border-image: url(:/image/Ico/ThreadPoolbuttonStop.ico);
	background:transparent; 
	color: rgb(167, 167, 167);
	border:4px groove white;
	border-radius:10px;
	padding:2px 4px;
}

QPushButton:pressed
{
	border-image: url(:/image/Ico/ThreadPoolbuttonStart.ico);
}
QPushButton:checked
{
	border-image: url(:/image/Ico/ThreadPoolbuttonStart.ico);
}
QToolTip{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="StartButton">
         <property name="minimumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>50</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton
{   
	
	border-image: url(:/image/Ico/TextbuttonStop.ico);
    background:transparent; 
}
QPushButton:hover
{
	border-image: url(:/image/Ico/TextbuttonStop.ico);
	color: rgb(167, 167, 167);
	border:4px groove white;
	border-radius:10px;
	padding:2px 4px;
}

QPushButton:pressed
{
	border-image: url(:/image/Ico/TextbuttonStart.ico);
}
QPushButton:checked
{
	border-image: url(:/image/Ico/TextbuttonStart.ico);
}
QToolTip{
	color: rgb(0, 0, 0);
	background-color: rgb(255, 255, 255);
}
</string>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="autoExclusive">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="TesttableWidget">
     <property name="minimumSize">
      <size>
       <width>100</width>
       <height>100</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
